optionals/variants_cup/config.cpp

```diff
@@ -0,0 +1,17 @@
+#include "script_component.hpp"
+
+class CfgPatches {
+    class ADDON {
+        name = COMPONENT_NAME;
+        units[] = {};
+        weapons[] = {};
+        requiredVersion = REQUIRED_VERSION;
+        requiredAddons[] = {"tacs_units", "CUP_AirVehicles_C130J"};
+        author = ECSTRING(main,Author);
+        authors[] = {"CUP Team", "GilleeDoo", "Tyrone"};
```

Comment:
```suggestion
        authors[] = {"GilleeDoo", "Tyrone"};
```
We don't put it in Arcadian either. It is mentioned in the `README.md` - stick to "authors of what was added in this mod".